CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.7.1
+
+Release date: August 30, 2017
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
+
+### Improvements and Bug Fixes
+
+#### Windows
+
+ - Client no longer freezes intermittently, such as when receiving desktop notifications. [#494](https://github.com/mattermost/desktop/issues/494), [#520](https://github.com/mattermost/desktop/issues/520)
```

Comment:
We had received reports only for Windows 10, but the improvement should perform for all clients. So I removed [Windows 10].